src/lang/english.txt

```diff
@@ -1277,9 +1277,9 @@ STR_CONFIG_SETTING_SUBSIDY_MULTIPLIER                           :Subsidy multipl
 STR_CONFIG_SETTING_SUBSIDY_MULTIPLIER_HELPTEXT                  :Set how much is paid for subsidised connections
 
 STR_CONFIG_SETTING_SUBSIDY_DURATION                             :Subsidy duration: {STRING2}
-STR_CONFIG_SETTING_SUBSIDY_DURATION_HELPTEXT                    :Set the number of years for which a subsidy is awarded
+STR_CONFIG_SETTING_SUBSIDY_DURATION_HELPTEXT                    :Set the number of {TKM years "12-minute periods"} for which a subsidy is awarded
```

Comment:
And now it becomes "12-minute periods" ... now you are just making shit up :P We went from "12 minutes" to "period" to now just combining the two ... please check your changes for this, and make it pretty :)